README.md

```diff
@@ -9,6 +9,20 @@ See the [Main Page](http://lamdu.org/)
 
 ## Installation
 
+### [NodeJS](https://nodejs.org/en/) & Build Time
+
+To drastically speed up Lamdu's installation under any OS, you can install
+`nodejs >= 6.2.1`<sup>**\***</sup> beforehand, such that `node` is in your `$PATH`.
+
+Enter `node -v` into terminal. If NodeJS is installed (and in your `$PATH`), this will print your current version. If it isn't, you'll get an error.
+
+If you do not install NodeJS, Lamdu's installation will build it from
+source.
+
+**<sup>\*</sup>** <u>*For Fedora Users*</u><b>&#x200A;:</b>
+&#x200A;Fedora packages have very long names. This may lead to some confusion.<br/>&nbsp;&nbsp;Consider `nodejs-1:6.11.2-1.fc25.x86_64`. This example indicates a NodeJS version of `6.11`, plus a little.<br/>&nbsp;&nbsp;The `-1:` is not a part of the version.
```

Comment:
This paragraph uses html-isms rather than markdown syntax (`<b>`, `&#x200A`, `<br/>`, `&nbsp;`).
It would probably be easier to maintain the doc using the markdown ways.

Reply:
Thanks for fixing. I was trying to apply both underlining and italics to some text, but the Markdown syntax for this combination did not work in Atom's Markdown preview. So I resorted to html-isms.

What is your setup for viewing rendered Markdown locally? (Or do you have one?)

Reply:
I'm editing in VS Code and using its markdown preview